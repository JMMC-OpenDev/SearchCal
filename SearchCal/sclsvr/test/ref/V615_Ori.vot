version https://git-lfs.github.com/spec/v1
oid sha256:fb4c8b304909eaaa89b31df4ec9ed1ae66a1edae8606307237274aa36e172ba9
size 210964
