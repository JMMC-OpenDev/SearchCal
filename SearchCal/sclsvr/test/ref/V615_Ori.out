version https://git-lfs.github.com/spec/v1
oid sha256:d48d503a273fc70136e7e14d71173e0771a3e368527cf84845d21d02a2820ef9
size 295222
